clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -1271,9 +1271,20 @@ private CompletedFetch initializeCompletedFetch(CompletedFetch nextCompletedFetc
             } else if (error == Errors.UNKNOWN_LEADER_EPOCH) {
                 log.debug("Received unknown leader epoch error in fetch for partition {}", tp);
             } else if (error == Errors.UNKNOWN_SERVER_ERROR) {
-                log.warn("Unknown error fetching data for topic-partition {}", tp);
+                log.warn("Unknown error fetching data while fetching offset "
```

Comment:
nit: for log messages, can we use the `{}` placeholders?

Reply:
Done